### DISCUSSION TITLE
DAI 2024-2025 - Java IOs benchmarking - Kilian Froidevaux

### DISCUSSION THREAD

**Discussion by kilianfroideva** [heig-vd-dai-course/heig-vd-dai-course#389] [Show and tell]
https://github.com/kilianfroideva/heig-vd-dai-course-java-ios-practical-content-.git
[benchmarks.md](https://github.com/user-attachments/files/17256334/benchmarks.md)

**Comment by ludelafo** [comment]
Thank you!

You can always find the solution in the solution repository if you ever need to compare your work with the official one here: https://github.com/heig-vd-dai-course/heig-vd-dai-course-solutions.

Do not hesitate to ask questions if something is not clear, either in this discussion or in the Q&A category of the GitHub Discussions here: https://github.com/orgs/heig-vd-dai-course/discussions/categories/q-a.
